src/cases/containers/restricted-case-access-container/restricted-case-access-container.component.ts

```diff
@@ -66,9 +68,30 @@ export class RestrictedCaseAccessContainerComponent implements OnInit, OnDestroy
 
   private getRestrictedCases(caseworkers: Caseworker[]): RestrictedCase[] {
     const restrictedCases: RestrictedCase[] = [];
-    this.idamIds.forEach((id) => {
+    this.idamIds.forEach(async (id) => {
       const user = caseworkers.find((caseworker) => caseworker.idamId === id);
       const caseRole = this.caseRoles.find((role) => role.actorId === id);
```

Comment:
This code worries me a bit. It finds one role, and if that has the role category judicial then it searches. I think it should be checking if any role has the role category judicial, not just the first one with the right actor Id

Reply:
I think this is a very poorly named variable as it really isn't caseRoles its an array of all the users that have access to the restricted case.
I have updated the variable name throughout this file to be usersWithAccess instead of caseRoles as i think that makes it clearer as to what the variable actually is.